iOSClient/Viewer/NCViewerMedia/NCPlayer/NCPlayerToolBar.swift

```diff
@@ -39,6 +40,13 @@ class NCPlayerToolBar: UIView {
     @IBOutlet weak var playbackSlider: UISlider!
     @IBOutlet weak var labelLeftTime: UILabel!
     @IBOutlet weak var labelCurrentTime: UILabel!
+    @IBOutlet weak var playerMessageHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var playerMessageProgressView: UIProgressView!
+    @IBOutlet weak var playerMessageTitle: UILabel!
+    @IBOutlet weak var playerMessageDescription: UILabel!
+    @IBOutlet weak var playerMessageDescriptionLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet weak var playerMessageImageView: UIImageView!
+    @IBOutlet weak var playerMessageButton: UIButton!
```

Comment:
I'm not sure the `NCPlayerToolBar` is the right place for this. It would be better to make a custom view class for the message.
Also, I don't like manipulating constrains at runtime like that. Did you consider using stack views to update the layout automatically?

Reply:
Was an experiment ... , but yes is better create a new class, the result was not what I wanted. [REMOVED]